packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -39,7 +39,7 @@ export const getPathOfNode = <TNode extends unknown>(
 };
 
 /**
- * Returns `true` if the expression is using `css` from `@compiled/react`.
+ * Returns `true` if the expression is using `css` from `@compiled/react` with a template literal.
```

Comment:
`with a template literal` is not quite correct, it should be `as a tagged template expression`

The naming of `isCompiledCSSTemplateLiteral` was always incorrect, since template literals refer to a different syntax.